app/controllers/concerns/announce_actions/announce_filter.rb

```diff
@@ -65,6 +65,7 @@ def decorator_for_model(model)
 
     def decorator_class(name)
       "Api::Msg::#{name}Representer".safe_constantize ||
+        "Api::Auth::#{name}Representer".safe_constantize ||
```

Comment:
yeah, if anything would help as wouldn't need to get auth version some special way.
I also looked through, there isn't an auth audio_file or image representer, so shouldn't break lambdas either.